src/install/memcached/windows/install-windows.mdx

```diff
@@ -0,0 +1,28 @@
+---
+componentType: default
+headingText: Configure the Memcached integration
+---
+
+1. In the Integrations directory, `C:\Program Files\New Relic\newrelic-infra\integrations.d\`, create a copy of the sample configuration file by running:
+
+```shell
```

Comment:
indent so theyre children of the step (?? unless it's the opposite from our indent sweep)

Reply:
actually u should explain this to me if i'm wrong lol i know it was a team meeting topic but i am doubting i processed that section correctly